Wrappers/Python/cil/plugins/astra/processors/FBP.py

```diff
@@ -63,20 +63,9 @@ class FBP(DataProcessor):
     def __init__(self, image_geometry=None, acquisition_geometry=None, device='gpu', **kwargs):
```

Comment:
Do we need the `kwargs`? As is users that pass `sinogram_geometry` or `volume_geometry` will not see any error, although if they don't pass the `acquisition_geometry` and `image_geometry` the instantiation will fail.

Reply:
I agree kwargs not needed here